spring/src/main/java/io/seata/spring/annotation/GlobalTransactionScanner.java

```diff
@@ -227,7 +227,7 @@ protected Object wrapIfNecessary(Object bean, String beanName, Object cacheKey)
                 }
 
                 LOGGER.info("Bean[{}] with name [{}] would use interceptor [{}]", bean.getClass().getName(), beanName, interceptor.getClass().getName());
-                if (!AopUtils.isAopProxy(bean)) {
+                if (!SpringProxyUtils.isProxy(bean) || !(bean instanceof SpringProxy)) {
```

Comment:
I think  `!SpringProxyUtils.isProxy` Contains the `!(bean instanceof SpringProxy))` condition.

Reply:
`!SpringProxyUtils.isProxy(bean) -> !AopUtils.isAopProxy(bean)  -> !((object instanceof SpringProxy &&
(Proxy.isProxyClass(object.getClass()) || ClassUtils.isCglibProxyClass(object.getClass()))))
-> !(bean instanceof SpringProxy)) || !((Proxy.isProxyClass(object.getClass()) || ClassUtils.isCglibProxyClass(object.getClass())))))
->   !(bean instanceof SpringProxy)) || (!Proxy.isProxyClass(object.getClass()) && !ClassUtils.isCglibProxyClass(object.getClass())))`

Reply:
`DubboUtil.isDubboProxyName(bean.getClass().getName()) || (**Proxy.class.isAssignableFrom(bean.getClass()**) || AopUtils.isAopProxy(bean))`. When **Proxy.class.isAssignableFrom(bean.getClass()** is true, it does not  containt `bean instanceof SpringProxy`